src/document/doc-builders.js

```diff
@@ -56,7 +56,8 @@ function indent(contents) {
 }
 
 /**
- * @param {number} n
+ * TODO: add missing types for paramter `n`
+ * @param {number | string | Doc} n
```

Comment:
Could we do something like this to be a little more consistent with the internal `makeAlign` function:

```suggestion
 * @param {number | string | Doc | false} n
```

Reply:
If the  documentation says `number | string` it probably mean `0` not `false`,  we should make it clear, but maybe later?

Reply:
yup